### ISSUE TITLE
Dark Mode support on Mac OS 10.14 Mojave

### ISSUE THREAD

**Issue by dpk** [aquamacs-emacs/aquamacs-emacs#141]
(Whoops. I accidentally hit return instead of tab when creating this issue, so if you subscribe to new issues by email you might not have got this text. Sorry!)

As far as I can tell, Aquamacs doesn’t support Dark Mode on Mac OS Mojave yet. Support would be nice to have!

**Comment by treese** [comment]
Can look at this after the update to Emacs 27 in Aquamacs 4.0.

**Comment by bestlem** [comment]
Isn't this just chosing a color theme - which has to be the old type as I think Aquamacs has broken the new themeing. There are several in color-theme.el
e.g.
    

```
(require 'color-theme)
(color-theme-initialize)
;; This is the dark theme I currently use
(color-theme-jsc-dark)
```

I can't see how you can code an actual dark mode that could be agreed on

**Comment by treese** [comment]
@bestlem Thanks for the note. I think there would be two parts to making a change like this: a default dark theme for Aquamacs, as you suggest, and the code to respond to the OS switching to dark for the system. The switching code would allow you to specify your own dark theme, of course. 

As noted, this won't happen for a while unless someone else wants to contribute the code.

**Comment by bestlem** [comment]
In that case IO would look at modus-themes https://gitlab.com/protesilaos/modus-themes it has switches between dark and light

A version of this is in emacs 28

But Aquamacs does need to fix being able to use the new style of theme

**Comment by nega0** [comment]
Aquamacs can use "new style" themes just fine, and has been able to for several versions.

**Comment by treese** [comment]
Yes, the new-style code has been there for a while. Some themes, like Modus, have a dependency on later versions of Emacs, and I hope to get past that later this year.
